lib/servicelog/railtie.rb

```diff
@@ -4,9 +4,12 @@
 
 module Servicelog
   class Railtie < ::Rails::Railtie
+    # app => Krakatoa::Application object
     initializer 'servicelog.insert_middlewares' do |app|
+      puts "\n\n....... Adding JobHeaders middleware to Servicelog\n\n"
       app.middleware.insert_after ActionDispatch::RequestId, Servicelog::StoreHeaders
       app.middleware.insert_after ActionDispatch::RequestId, Servicelog::RequestId
+      app.middleware.insert_after ActionDispatch::RequestId, Servicelog::JobHeaders
```

Comment:
Do you really need to register this middleware here? The middleware should be registered as documented here in the application: https://github.com/phstc/shoryuken/wiki/Middleware#registering-a-global-middleware

I do not think that this is necessary for this specific middleware

Reply:
I've removed this change as the middleware will be registered to Shoryuken from each app using the gem, for KK:

https://github.com/volcanic-uk/krakatoa/pull/1781/files#diff-5f15d419ef48d4e547e23fe2f2460f20R9

Reply:
Please a new section in the readme (https://github.com/volcanic-uk/servicelog/blob/master/README.md) explain how to set up it on an app.

Reply:
Readme updated with new section:

<img width="672" alt="Screenshot 2020-10-06 at 11 35 18" src="https://user-images.githubusercontent.com/20913661/95184747-1ad5b980-07c8-11eb-967a-6f491bb5bb73.png">